js-jest/test/gilded_rose.test.js

```diff
@@ -1,20 +1,39 @@
-const {Shop, Item} = require("../src/gilded_rose");
+const {Shop, Item, NormalItem, ConjuredItem, AgedBrie,
+  LegendaryItem, ConcertTickets} = require("../src/gilded_rose");
 
 test("First Normal Item Test ", function() {
```

Comment:
Now that you've done the refactor to make this Open-Closed, these tests seem more like integration tests to me and less like unit tests. I say this because they test not only the Shop class, but also the functionality you've encapsulated inside each of the item type classes.